common-bsp/recipes-kernel/device-tree-overlays/device-tree-overlays_git.bb

```diff
@@ -0,0 +1,33 @@
+DESCRIPTION = "Device Tree Overlays for bb.org boards"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/LICENSE;md5=4d92cd373abda3937c2bc47fbc49d690"
+
+COMPATIBLE_MACHINE = "(beaglebone)"
+
+PV = "1.0+git${SRCPV}"
+PR = "r1"
```

Comment:
Sounds reasonable. However I couldn't find any documentation mentioning this `PR` deprecation.